03/styles/global.css

```diff
@@ -25,4 +25,85 @@
     border-color: #baffc9;
 }
 
-/* .mobile.css */
\ No newline at end of file
+
+
+body {
+    display:grid;
+    grid-template-areas: 
+    "picture-area"
+    "text-area"
+    "video-area"
+    ;
+}
+
+.p {
+    grid-area: picture-area;
+  
+    
+}
+
+.t {
+    grid-area: text-area;
+    
+}
+
+.v {
+    grid-area: video-area;
+    
+}
+
+
+
+@media (min-width:599px){
```

Comment:
Z HTML wynika, że przedziału definiujesz do plików, a nie w kodzie CSS.
Wybierz jedno wygodne dla siebie rozwiązanie.

Zastanawiam się dlaczego też ustawiasz to samo co już zrobiłeś w linii 30 -37